flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/hive/GravitinoHiveCatalogFactory.java

```diff
@@ -81,4 +86,39 @@ public Set<ConfigOption<?>> requiredOptions() {
   public Set<ConfigOption<?>> optionalOptions() {
     return hiveCatalogFactory.optionalOptions();
   }
+
+  /**
+   * Define gravitino catalog provider.
```

Comment:
Could you complete the related java doc?

Reply:
added